wpeverest/everest-forms
# Feature- Review notice

### All Submissions:






### Changes proposed in this Pull Request:


This PR is about asking the users for review on WordPress.org. If the user is using Everest Forms for more than 15 days then a notice will appear. If the user will give the rating or if he chooses I already did then the notice won't appear again for that user and if the clicks on maybe later then after 15 days the notice will come again.

### How to test the changes in this Pull Request:

1. Switch to the branch and deactivate and activate the plugin if its already activated.
2. Now set to date 15 days ahead. The notice will appear.
3. And do the rest as mentioned above.

### Types of changes:




### Other information:




### Changelog entry

> Feature - Review notice
